src/relay/backend/build_module.cc

```diff
@@ -438,13 +439,14 @@ class RelayBuildModule : public runtime::ModuleNode {
 
     auto lowered_funcs = graph_codegen_->GetLoweredFunc();
     if (lowered_funcs.size() == 0) {
-      LOG(WARNING) << "no lowered funcs exist in the compiled module";
+      ret_.mod = tvm::codegen::CSourceModuleCreate("", "");
```

Comment:
Sorry for the back and forth. Could you please add a comment here so that ppl would know what we are doing here?

Reply:
You can force push so that your previous CI could be terminated earlier.

Reply:
Sure thing, just added that